roles/management-cluster/tasks/main.yml

```diff
@@ -75,3 +75,4 @@
     host: "{{ vm_public_network_ip }}"
     port: 6443
     delay: 60
+    timeout: 3600 # Adding a timeout to allow deployment on machines with slower disks
```

Comment:
We're going from 5 minutes to 1 hour, how long does it actually take in your environment?  

If we're going to hard-code I'd suggest a lower value (600 seconds e.g twice the current default?) or better add a variable to `roles/management-cluster/defaults/main.yml` so the value can be adjusted (it's fine to make the default some larger number though, e.g 600 but probably not an hour)

Reply:
I didn't test all possible values, I tried 600, 1200 and then 3600.
1200 was not enough for the machine :(
I will look into creating it as a variable then. Thanks!